### ISSUE TITLE
Create UserSnap for gov.tools to collect feedback

### ISSUE THREAD

**Issue by bosko-m** [IntersectMBO/govtool#1860]
### Area

Wrapper

### Is there new design needed?

No

### What?

We should include usersnap feedback collection in the production (mainnet) version of the GovTool.

### Why?

To be able to collect feedback from the user and put it in the transparently available community backlog.

### How?

Include the specific code generated in Usersnap admin dashboard and assign it to the feedback button in the footer.

**Comment by MSzalowski** [comment]
That issue shouldn't be needed. At least as `creation`. If Usersnap is not working on gov.tools - that means that env keys are not set.

@mpavlovicbb Is `VITE_USERSNAP_SPACE_API_KEY` env set on gov.tools?

**Comment by Ryun1** [comment]
https://github.com/IntersectMBO/govtool/issues/1864

**Comment by bosko-m** [comment]
True @Ryun1 . What we need is to have usersnap button on preview and mainnet tied to Intersect account.
But its fully ok to go and release the GovTool the way its set up now and move this issue to post release, so Im moving it to another Milestone.

**Comment by l-br1** [comment]
was this connected to the Intersect account and also the automation to add tickets to github also added back ?

**Comment by l-br1** [comment]
yes, usersnap works, did we confirm it is connected to our account and not the Byron one?

**Comment by Ryun1** [comment]
> did we confirm it is connected to our account and not the Byron one?

looks like Byron

**Comment by l-br1** [comment]
ok, can we please re-open this task? The action was to connect it to the Intersect Usersnap account. @kickloop can give access to set it up

**Comment by MSzalowski** [comment]
That is not on the Byron side, as the whole usersnap integration (in-code) is already done.

The only thing is to provide the appropriate `VITE_USERSNAP_SPACE_API_KEY` in qovery (on the infra level)

**Comment by l-br1** [comment]
@kickloop when you can we need to provide the byron team either access or the code :)
